From 076ff1b88ac0407cd08fa13caaf018198825dd96 Mon Sep 17 00:00:00 2001
From: Lin Fuzhen <fuzhen.lin@freescale.com>
Date: Wed, 17 Oct 2012 11:25:05 +0800
Subject: [PATCH 354/498] ENGR00229943 PowerHAL: change the target folder to
 /system/lib/hw

Change the build out target folder to system/lib/hw

Signed-off-by: Lin Fuzhen <fuzhen.lin@freescale.com>
---
 power/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/power/Android.mk b/power/Android.mk
index f86b50a..970c135 100644
--- a/power/Android.mk
+++ b/power/Android.mk
@@ -20,7 +20,7 @@ LOCAL_PATH := $(call my-dir)
 # hw/<POWERS_HARDWARE_MODULE_ID>.<ro.hardware>.so
 include $(CLEAR_VARS)
 
-LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR_SHARED_LIBRARIES)/hw
+LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
 LOCAL_SHARED_LIBRARIES := liblog libcutils
 LOCAL_SRC_FILES := power.c
 LOCAL_MODULE := power.$(TARGET_BOARD_PLATFORM)
-- 
1.8.0

